### ISSUE TITLE
small fixes for new and old maps

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#277]
```
here are some small fixes for new, old and very old maps.
- sometimes the player stood above objects
- some minor glitches
- trees and other things that you couldn't walk under until now
- etc.
```

Original issue reported on code.google.com by `SamuelPl...@gmail.com` on 18 Nov 2011 at 1:03

Attachments:
- [small map fixes (trees, move on objects, etc).patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-277/comment-0/small map fixes %28trees, move on objects, etc%29.patch)

**Comment by GoogleCodeExporter** [comment]
```
BTW: the new maps look great.
```

Original comment by `SamuelPl...@gmail.com` on 18 Nov 2011 at 1:04

**Comment by GoogleCodeExporter** [comment]
```
Updated Patch File attached:
```

Original comment by `SamuelPl...@gmail.com` on 19 Nov 2011 at 2:42

Attachments:
- [small map fixes (trees, move on objects, etc).patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-277/comment-2/small map fixes %28trees, move on objects, etc%29.patch)

**Comment by GoogleCodeExporter** [comment]
```
The majority of these changes are now committed in r219. Thanks Samuel.
```

Original comment by `oskar.wi...@gmail.com` on 13 Feb 2012 at 9:33
- Changed state: **Ready**
- Added labels: **Milestone-0.6.11**

**Comment by GoogleCodeExporter** [comment]
```
For future reference, this is the changes from comment 2 in human-readable text 
form.
```

Original comment by `oskar.wi...@gmail.com` on 13 Feb 2012 at 10:30

Attachments:
- [small map fixes (trees, move on objects, etc).text.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-277/comment-4/small map fixes %28trees, move on objects, etc%29.text.patch)

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.11. Closing issue.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2012 at 9:31
- Changed state: **Fixed**
